pkg/gitreceive/storage/slug_builder_info.go

```diff
@@ -1,10 +1,6 @@
 package storage
 
-import (
-	"fmt"
-
-	"github.com/deis/builder/pkg/gitreceive/git"
-)
+import "fmt"
```

Comment:
@kmala can you still put imports in parentheses, even if there is one? This one would be the following:

```go
import (
	"fmt"
)
```